src/SVGSkin.js

```diff
@@ -25,19 +26,22 @@ class SVGSkin extends Skin {
         /** @type {WebGLTexture} */
         this._texture = null;
 
-        /** @type {number} */
-        this._textureScale = 1;
+        /** @type {Array.<SVGMIPs>} */
+        this._scaledMIPs = [];
+
+        /** @type {Number} */
+        this._maxTextureScale = 1;
 
         /** @type {Number} */
-        this._maxTextureScale = 0;
+        this._largestTextureScale = 1;
     }
 
     /**
      * Dispose of this object. Do not use it after calling this method.
      */
     dispose () {
         if (this._texture) {
-            this._renderer.gl.deleteTexture(this._texture);
+            this._renderer.gl.deleteTexture(this._texture.getTexture());
```

Comment:
Shouldn't `SVGSkin.dispose` call `gl.deleteTexture` on *all* `SVGMIP` textures, instead of just the current one?

Reply:
Oh good point! I've updated `dispose` to delete all of the textures and reset `_scaledMIPs` and `_largestTextureScale`. Let me know if I'm missing anything else.